cpp/core/utils/TaskContext.cc

```diff
@@ -55,13 +52,7 @@ void bindToTask(std::shared_ptr<void> object) {
     taskContextStorage->bind(object);
     return;
   }
-  // The fallback storage is used. Spark sometimes creates sub-threads from a task thread. For example,
-  //   PythonRunner.scala:183 @ Spark3.2.2
-  //   GlutenSubqueryBroadcastExec.scala:73 @6bce2c33
-  std::lock_guard<std::mutex> guard(fallbackStorageMutex);
-  std::cout << "Binding a shared object to fallback storage. This should only happen on sub-thread of a Spark task. "
-            << std::endl;
-  fallbackStorage->bind(object);
+  GLUTEN_CHECK(false, "[BUG] Found untracked computation, please submit issue with query plan.");
```

Comment:
This fix is not related to the Java side [TaskContext, [TID, Resource]] design?

Reply:
Just put check here to find corner cases we ignored.